src/go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/go-openapi/validate v0.19.10
 	github.com/go-redis/redis/v8 v8.11.4
 	github.com/go-sql-driver/mysql v1.5.0
+	github.com/gocarina/gocsv v0.0.0-20210516172204-ca9e8a8ddea8
```

Comment:
why don't we choose the "encoding/csv"?

Reply:
Deliberately chose the current `gocarina/gocsv` library over the `encoding/csv` since it sis supports binding CSV fields to the struct fields during definition phase itself. For reference please take a look at how the `export.Data` struct is defined with the CSV annotations. This is not possible in `encoding/csv`  resulting in more verbose code